src/runtime_src/core/pcie/driver/linux/xocl/xocl_fdt.h

```diff
@@ -80,6 +80,8 @@
 #define NODE_P2P "ep_p2p_00"
 #define NODE_DDR4_RESET_GATE "ep_ddr_mem_srsr_gate_00"
 #define NODE_ADDR_TRANSLATOR "ep_remap_data_c2h_00"
+#define NODE_MAILBOX_XRT "ep_mailbox_xrt_00"
+#define NODE_OSPI_CACHE "ep_ospi_cache_00"
```

Comment:
Please contact Adam to make sure he aware these new endpoints.

Reply:
Mark will contact Adam and make sure those new endpoints are ok.